packages/flutter/lib/src/services/text_input.dart

```diff
@@ -609,6 +611,50 @@ class TextInputConfiguration {
   /// {@endtemplate}
   final bool enableIMEPersonalizedLearning;
 
+  /// {@template flutter.services.TextInputConfiguration.contentCommitMimeTypes}
+  /// Used when a user inserts image-based content through the device keyboard
+  /// on Android only.
+  ///
+  /// The passed list of strings will determine which MIME types are allowed to
+  /// be inserted via the device keyboard. If no `onContentCommitted` callback
+  /// is provided, this field will be ignored.
+  ///
+  /// Defaults to ['image/png', 'image/bmp', 'image/jpg', 'image/tiff', 'image/gif', 'image/jpeg', 'image/webp']
```

Comment:
I'm not sure what the _best_ way to do it is, however, I know MIME Types can be wild-carded, like so: `image/*`, so if we limit it to an ENUM, we would eliminate that from being used (unless we built in an entry for that). 

If that's something you think we should do, we can implement it. Otherwise, I don't think it's a big deal to use strings here